docs/core-concepts/external-routes.md

```diff
@@ -0,0 +1,11 @@
+# External Routes
```

Comment:
Do you feel strongly its not a core concept? It reads very nicely to go from routes to external routes. 

As far as meta vs state. I'd rather describe what they are rather than be descriptive on how to use them. Which I think we can do in the route docs.

Reply:
I understand not wanting to bloat docs with how we think you should use things. 

I'm in favor of getting users to basic understanding as quickly and simply as possible. Majority of users will probably never use external routes.